python/tox.ini

```diff
@@ -10,7 +10,7 @@ commands =
     python test.py
 
 [testenv:qa]
-basepython = python3.6
+basepython = python3
```

Comment:
The problem with it is that it breaks `travis ci` unfortunately (https://github.com/ccxt/ccxt/blob/master/.travis.yml). I mean, it was `python3` previously, but then upon some version switch in some of the deps – the default python3 on Travis stopped working (it defaulted to 3.4 which is deprecated by now), so i had to switch to `python3.6` to workaround that temporarily, just didn't have a chance to dive deeper into it at that time. So, this change might cause a consequent breakup and we might need to properly fix Travis first.

Reply:
Symlink ```python3.6``` to ```python3``` in Travis ?

Reply:
@xCuri0 yep, i've been thinking in the same direction, just didn't have the time to actually set up the right paths in the Travis script. Will resolve it today.

Reply:
Heh it has self-resolved %)